source/write.txt

```diff
@@ -4,10 +4,143 @@
 Write Data to MongoDB
 =====================
 
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :description: Learn how to use the {+driver-short+} to write data to MongoDB.
+   :keywords: usage examples, save, crud, create, code example
+
 .. toctree::
    :titlesonly:
    :maxdepth: 1
 
    /write/insert
    /write/update
    /write/delete
+
+.. TODO:
+   /write/replace
+   /write/bulk-write
+   /write/gridfs
+
+Overview
+--------
+
+On this page, you can see copyable code examples that show common
+{+driver-short+} methods for writing data to MongoDB.
+
+.. tip::
+
+   To learn more about any of the methods shown on this page, see the link
+   provided in each section.
+
+To use an example from this page, copy the code example into the
+:ref:`sample application <cpp-write-sample>` or your own application.
+Be sure to replace all placeholders, such as ``<connection string>``, with
+the relevant values for your MongoDB deployment.
+
+.. _cpp-write-sample:
+
+.. include:: /includes/usage-examples/sample-app-intro.rst
+
+.. literalinclude:: /includes/usage-examples/sample-app.cpp
+   :language: cpp
+   :dedent:
+   :linenos:
+   :emphasize-lines: 23-25
+
+Insert One
+----------
+
```

Comment:
S: I think it's good practice to always introduce code examples. See my PR for kotlin sync for intros you can copy over https://github.com/mongodb/docs-kotlin-sync/pull/5
Applies to all sections